src/lib.rs

```diff
@@ -732,6 +736,16 @@ impl Queue {
         }
     }
 
+    /// Set whether the recv function blocks or not
+    pub fn set_nonblocking(&mut self, non_blocking: bool) {
+        if non_blocking {
+            self.recv_flag = MSG_DONTWAIT;
+        }
+        else {
```

Comment:
The format here is wrong. Have you run `cargo fmt`?

Reply:
No, I didn't run that. I'll go and format it